drivers/SmartThings/matter-sensor/fingerprints.yml

```
@@ -16,6 +16,11 @@ matterManufacturer:
     vendorId: 0x130a
     productId: 0x004D
     deviceProfileName: contact-battery
+  - id: "Eve/Weather"
+    deviceLabel: Eve Weather
+    vendorId: 0x130a
+    productId: 0x57
```

nit: can you add the leading zeroes for consistency with other fingerprints in this file?
```suggestion
    productId: 0x0057
```

---

Done!